Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The deliverables of this project will be split into to two sections, physical and non-phsyical.
+
+\subsubsection{Physical Deliverables}
+\label{subsubsec: physical deliverables}
+
+The physical deliverables of this project with the CGT\@. This includes the propulsion system and the test stand. The propulsion system will include a regulator,
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
+consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
+The system will also be displayed at Imagine RIT 2018. This will include a full poster and data representation.
+
+\subsubsection{Non-physical Deliverables}
+\label{subsubsec: non-physical deliverables}
+
+The non-physical deliverables of this project will be the project culmination report which will include a very detailed examination of the projects
+results. These results include the expected performance versus the test performance, the nozzle dimension polynomial approximation, the structure performance, and the
+propulsion system performance. The structure performance is basically a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+
+There is a preliminary plan for the semester that is displayed in further detail in Appendix A. The following list will display the description and timing
+of the different phases; design, sanity tests, experimentation, and data analysis.
+
+\subsubsection{Design Phase}
+\label{subsubsec: design phase}
+    This will be the first phase and reappearing after the culmination of the experimentation phase.
+    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+
+    \textit{Projected End Date: Week 4 (maybe 5 if schedule slips, but no later)}
+
+\subsubsection{Build \& Preliminary Test Phase}
+\label{subsubsec: sanity test phase}
+    This phase is concerned with the building of the system. The test stand is already built. However a few revisions might be applied, then the team will need
+    to re-build to the new design. The main part of the building stage will be the construction of the propulsion system. This has not been built yet therefore, that
```

Comment:
This is not clear. Does this mean the existing test stand will receive minor modifications, or will be entirely rebuilt?